themes/gatsby-theme-blog/src/components/post-footer.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { css } from "theme-ui"
+import PostFooterContent from "./post-footer-content"
+
+const Footer = props => (
+  <footer
+    css={css({
+      mt: 4,
+      pt: 3,
+    })}
+  >
+    <PostFooterContent {...props} />
```

Comment:
Open to whatever here, but it seems like it wouldn't be too difficult to shadow things like this if footer & footer-content were combined into a single module/component

Reply:
Yeah, it's just the tradeoff. I was trying to minimize the surface a person would have to change, to simply update the content.